charts/milvus/templates/config.tpl

```diff
@@ -31,7 +31,7 @@ etcd:
 {{- else }}
   endpoints:
 {{- range $i := until ( .Values.etcd.replicaCount | int ) }}
-  - {{ $etcdReleaseName }}-{{ $i }}.{{ $etcdReleaseName }}-headless.{{ $namespace }}.svc.cluster.local:{{ $etcdPort }}
+  - {{ $etcdReleaseName }}-{{ $i }}.{{ $etcdReleaseName }}-headless.{{ $namespace }}.svc.{{ .Values.etcd.clusterDomain }}:{{ $etcdPort }}
```

Comment:
inside a `range` loop, Values should be referenced by  {{$.Values.xxx}}

ref: https://helm.sh/docs/chart_template_guide/variables/

Reply:
fixed